distributed/cli/tests/test_dask_worker.py

```diff
@@ -264,9 +264,14 @@ def test_bokeh_non_standard_ports(loop):
                     assert response.ok
                     redirect_resp = requests.get("http://127.0.0.1:4833/main")
                     redirect_resp.ok
+                    # TEST PROXYING WORKS
+                    url = "http://127.0.0.1:8787/proxy/4833/127.0.0.1/status"
+                    response = requests.get(url)
+                    assert response.ok
                     break
                 except Exception:
                     sleep(0.5)
                     assert time() < start + 20
+
```

Comment:
If possible, lets also test with an async test.  The CLI tests are the least pleasant to work with when things go wrong and we have to debug.  There is a sort of hierarchy of `gen_cluster` > `c, s, a, b` fixtures > CLI for distributed tests.  See https://distributed.dask.org/en/latest/develop.html#writing-tests for more information.

The file `distributed/bokeh/tests/test_scheduler_bokeh.py` should have some good examples to pull from.

Reply:
done